src/xcode/ENA/ENA/Source/Services/CCLService/__tests__/ccl-text-descriptor-test-cases.json

```diff
@@ -533,6 +533,39 @@
 		  "text": "vor 2 Minuten"
 		}
 	  ]
+	},
```

Comment:
Can you please update the entire file with the content from https://github.com/corona-warn-app/cwa-app-ccl/releases/download/latest/ccl-text-descriptor-test-cases.gen.json so that the date and the treeish in the header are updated, too?

Reply:
see https://github.com/corona-warn-app/cwa-app-ios/pull/4202/commits/9dd3dc32531080667839ff5dcc98f42c49065c7a

Reply:
👍